Frameworks/CoreLocation/CoreLocationFunctions.mm

```diff
@@ -18,19 +19,24 @@
 #import <CoreLocation/CoreLocationFunctions.h>
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
-/*BOOL CLLocationCoordinate2DIsValid(CLLocationCoordinate2D coord) {
-    UNIMPLEMENTED();
-    return StubReturn();
-}*/
+BOOL CLLocationCoordinate2DIsValid(CLLocationCoordinate2D coord) {
```

Comment:
This is treated as 0s when null, so I can't actually check if this is NULL/nullptr or anything.